README.md

```diff
@@ -58,9 +58,13 @@ See section [recursion](/recursion) for description.
 
 *(Coming soon)*
 
-## Closures
+## [Closures](/closures)
 
-*(Coming soon)*
+Closures are functions that scope exists within a variable. Any function defined within a closure has access to the scope and variables created
+in the parent function. 
```

Comment:
"functions that scope exists within a variable" -- doesn't really mean anything :)  The concept of closure is the concept of letting a function have access to things that are scoped around it, and tying the lifetime of those things to the lifetime of the function.  Basically (sort of) creating a snapshot of the call stack and storing it along with the function, when you create one.